Microsoft.OData.AzureFunctions/ODataValueProvider.cs

```diff
@@ -6,19 +6,17 @@ namespace Microsoft.OData.AzureFunctions
 {
     public class ODataValueProvider<T> : IValueProvider
     {
-        private HttpRequest request;
-        private IEdmModelProvider modelProvider;
-        private Type clrType;
-        private string routePrefix;
+        private readonly HttpRequest request;
+        private readonly IEdmModelProvider modelProvider;
+        private readonly string routePrefix;
 
-        public ODataValueProvider(HttpRequest request, IEdmModelProvider modelProvider, Type clrType)
+        public ODataValueProvider(HttpRequest request, IEdmModelProvider modelProvider)
         {
             this.request = request;
             this.modelProvider = modelProvider;
-            this.clrType = clrType;
             this.routePrefix = "api";
```

Comment:
We had agreed with Kennedy that the route prefix should be passed from the Azure function `OData` attribute...

Reply:
@gathogojr We can fix that later. There are more edge cases to consider

Reply:
@KenitoInc Of all the edge cases I can think of, the one of route prefix is the one that can bring down everything like a house of cards. If it's changed from _host.json_ file like in the sample below, everything breaks since the consumer of the library has no way of passing the route prefix to the library
```json
{
    "version": "2.0",
  "logging": {
    "applicationInsights": {
      "samplingSettings": {
        "isEnabled": true,
        "excludedTypes": "Request"
      }
    }
  },
  "extensions": {
    "http": {
      "routePrefix": "SOMETHING_ELSE_INCLUDING_EMPTY_STRING"
    }
  }
}
```